features/FEATURE_LWIP/lwip-interface/lwipopts.h

```diff
@@ -80,10 +80,12 @@
 #define DEFAULT_RAW_RECVMBOX_SIZE   8
 #define DEFAULT_ACCEPTMBOX_SIZE     8
 
+#ifndef TCPIP_THREAD_STACKSIZE
```

Comment:
Should wait for #4048. If it's merged, target options can go to lwip's mbed_lib.json and lwipopts.h can be left untouched.
`
+        },
+        "tcpip-thread-stacksize": {
+            "help": "Stack size for lwip TCPIP thread",
+            "value": 1200
+        },
+        "default-thread-stacksize": {
+            "help": "Stack size for lwip system threads",
+            "value": 512
+        }
+    },
+    "target_overrides": {
+        "REALTEK_RTL8195AM": {
+            "tcpip-thread-stacksize": 1600
`